src/etl/lambdas/etl_task_file_copy/copy_ftp.py

```diff
@@ -37,9 +37,10 @@ def get_ftp(location):
                 source_file = lf[0]["name"]
             else:
                 source_file = None
-
+        fileResult = { "fileFound": True, "fileName": source_file }
```

Comment:
Boolean value **True** should be lowercase, right?

Reply:
In Python they are mixed case

Reply:
I've messed this up a billion times in JS

Reply:
Thank you! My brain just assumes JS and zero'd in on that not even realizing I've got Python code in front of me 🤦‍♂️😂